autoscale/src/main/resources/application.yml

```diff
@@ -44,6 +45,19 @@ periscope:
       addr: localhost
       port: 5432
   cloudbreak.url: http://localhost:9091
+  enabledAutoscaleMonitors:
+    time-monitor:
+      enabled: true
+    load-monitor:
+      enabled: true
+    rejected-thread-monitor:
+      enabled: true
+    cluster-manager-host-health-monitor:
+      enabled: true
```

Comment:
cluster-manager-host-health-monitor is required for periscope functionality, it removes orphaned clusters in periscope which are deleted in cloudbreak.

Reply:
It seems to do this by talking to the cluster itself, instead of getting the state from Cloudbreak? Letting Cloudbreak talk to the cluster for health, and then getting this information from Cloudbreak I think will work better. Future enhancement: Not related to this change at all.

Reply:
Yes will create jira for that.